### PR TITLE
TSA-126: Basic permissions model

### PR DESCRIPTION
**Pull request by robyoung-digital**

### CODE REVIEW

File: thunderstorm_auth/flask.py [hunk 1]
```diff
@@ -67,14 +86,170 @@ def _get_jwks():
...
+        _list_permissions(db_session, permission_model)
+    )
+    group.command(name='update')(
+        _update_permissions(app, db_session, permission_model)
+    )
+
+
+def _permissions():
+    """Manage auth permissions"""
+
```
Comment (shipperizer) [inline thunderstorm_auth/flask.py]: do we need a `pass` in here?

**Comment by shipperizer** [comment]
Next change we do to this lib is to make it a flask extension, we'll have to drag in a falcon expert to make a correspondent one

File: thunderstorm_auth/flask.py (hunk 1 above)
Comment (robyoung-digital) [reply thunderstorm_auth/flask.py]: no, because it has a docstring, but we also don't need the function at all 🙄

Comment (robyoung-digital) [reply thunderstorm_auth/flask.py]: oh no, click needs it because command groups have to be functions

**Pull request merged**